charts/matrix-stack/configs/matrix-rtc/sfu/config-overrides.yaml.tpl

```diff
@@ -23,6 +23,16 @@ logging:
 
 # WebRTC configuration
 rtc:
+  use_external_ip: {{ .useStunToDiscoverPublicIP }}
```

Comment:
I think this should go into `config-underrides.yaml.tpl` for at least 1 monthly release cycle. This would normal be the correct place for it but given people may be configuring this manually for now, I think we should respect their manual configurations temporarily to give them time to upgrade